pyuvdata/utils.py

```diff
@@ -1059,9 +1061,1341 @@ def unphase_uvw(ra, dec, uvw):
+        Apparent declination of the previous phase center, required if not deriving
+        baseline coordinates from antenna positions and from_enu=False. Shape is
+        (Nblts,).
+    lst_array : ndarray of float
+        Local apparent sidereal time, required if deriving baseline coordinates from
+        antenna positions, or converting to/from ENU coordinates. Shape is (Nblts,).
+    use_ant_pos : bool
+        Switch to determine whether to derive uvw values from the antenna positions
+        (if set to True), or to use the previously calculated uvw coordiantes to derive
+        new the new baseline vectors (if set to False). Default is True.
+    uvw_array : ndarray of float
+        Array of previous baseline coordinates (in either uvw or ENU), required if
+        not deriving new coordinates from antenna positions.  Shape is (Nblts, 3).
+    antenna_positions : ndarray of float
+        List of antenna positions relative to array center in ECEF coordinates,
+        required if not providing `uvw_array`. Shape is (Nants, 3).
+    antenna_numbers: ndarray of int
+        List of antenna numbers, ordered in the same way as `antenna_positions` (e.g.,
+        `antenna_numbers[0]` should given the number of antenna that resides at ECEF
+        position given by `antenna_positions[0]`). Shape is (Nants,), requred if not
+        providing `uvw_array`. Contains all unique entires of the joint set of
+        `ant_1_array` and `ant_2_array`.
+    ant_1_array : ndarray of int
+        Antenna number of the first antenna in the baseline pair, for all baselines
+        Required if not providing `uvw_array`, shape is (Nblts,).
+    ant_2_array : ndarray of int
+        Antenna number of the second antenna in the baseline pair, for all baselines
+        Required if not providing `uvw_array`, shape is (Nblts,).
+    telescope_lat : float
+        Latitude of the phase center, units radians, required if deriving baseline
+        coordinates from antenna positions, or converting to/from ENU coordinates.
+    telescope_lon : float
+        Longitude of the phase center, units radians, required if deriving baseline
+        coordinates from antenna positions, or converting to/from ENU coordinates.
+    from_enu : boolean
+        Set to True if uvw_array is expressed in ENU coordinates. Default is False.
+    to_enu : boolean
+        Set to True if you would like the output expressed in EN coordinates. Default
+        is False.
+
+    Returns
+    -------
+    new_coords : ndarray of float64
+        Set of baseline coordinates, shape (Nblts, 3).
+    """
+    if to_enu:
+        if lst_array is None and not use_ant_pos:
+            raise ValueError(
+                "Must include lst_array to calculate baselines in ENU coordinates!"
+            )
+        if telescope_lat is None:
+            raise ValueError(
+                "Must include telescope_lon to calculate baselines"
+                "in ENU coordinates!"
+            )
```

Comment:
I think this message should mention `telescope_lat` rather than `telescope_lon` since that's the variable that's being checked here.

Reply:
Fixed!